academy/ibc/ica.md

```diff
@@ -0,0 +1,610 @@
+# Interchain Accounts
+
+Another application module [implemented in the IBC-go](https://github.com/cosmos/ibc-go/tree/main/modules/apps/27-interchain-accounts) repository is the [Interchain Accounts(ICS-27)](https://github.com/cosmos/ibc/blob/master/spec/app/ics-027-interchain-accounts/README.md). 
+
+**Interchain Accounts(ICA)** allows you to control an account on a **host chain** from a **controller chain**.
+
+![ICA Overview](images/icaoverview.png)
+
+**Host Chain**: the chain where the interchain account is registered. The host chain listens for IBC packets from a controller chain which should contain instructions (e.g. cosmos SDK messages) which the interchain account will execute.
+
+**Controller Chain**: the chain that registers and controls an account on a host chain. The controller chain sends IBC packets to the host chain to control the account. A controller chain must have at least one interchain accounts authentication module in order to act as a controller chain.
+
```

Comment:
```suggestion
<HighlightBox type="info">

The Interchain Accounts application module is structured to support the ability of exclusively enabling controller or host functionality. This can be achieved by simply omitting either controller or host `Keeper` from the Interchain Accounts `NewAppModule` constructor function, and mounting only the desired submodule via the `IBCRouter`. Alternatively, submodules can be enabled and disabled dynamically using on-chain parameters.

</HighlightBox>
```